test/prob/exponential/exponential_test.hpp

```diff
@@ -21,10 +21,10 @@ class AgradDistributionsExponential : public AgradDistributionTest {
     parameters.push_back(param);
     log_prob.push_back(-57.13902344686439249699);  // expected log_prob
 
-    param[0] = 1e-08;
+    param[0] = 1e-2;
```

Comment:
This test point needs to be far enough away from zero so that the finite difference checks don't go out of range.